### PR TITLE
Increase nginx client max body size, split size legacy migration

### LINKED ISSUE #1405: 🐛 [Bug] 413 Request Entity too large
> [datenbrei](https://github.com/datenbrei) **Authored by [datenbrei](https://github.com/datenbrei)**
_<time datetime="2019-08-27T14:31:04Z" title="Tuesday, August 27th 2019, 4:31:04 pm +02:00">Aug 27, 2019</time>_
_Closed <time datetime="2019-08-28T12:36:22Z" title="Wednesday, August 28th 2019, 2:36:22 pm +02:00">Aug 28, 2019</time>_
---

## :bug: Bugreport

Bigger photos give an error

### Steps to reproduce the behavior
1. Write a new post
2. Add an bigger image (here 1,7 MB)
3. See the Error:
bild-zu-gross

### Expected behavior

It should be clear and statet what the biggest photo is, that can be uploaded, a clear limit.

### Version & Environment
 Type: [Desktop]      
 - OS: [Linux]      
 - Browser: [Firefox] 
 - Version [68.0.1]  
 - Device: [Notebook]

### PR DESCRIPTION
**Pull request by Tirokk**
> [mattwr18](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-27T16:05:55Z" title="Tuesday, August 27th 2019, 6:05:55 pm +02:00">Aug 27, 2019</time>_
_Merged <time datetime="2019-08-28T17:17:05Z" title="Wednesday, August 28th 2019, 7:17:05 pm +02:00">Aug 28, 2019</time>_
---

- allows the client to send files at least the size we agreed in the vue-dropzone
- we don't need such a small split size as we don't run it on the remote server

### Issues

- fixes #1405

### CODE REVIEW

**Comment by Tirokk** [comment]
> [mattwr18](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-27T22:08:01Z" title="Wednesday, August 28th 2019, 12:08:01 am +02:00">Aug 28, 2019</time>_

---


learned about kubernetes and annotations here
https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#custom-max-body-size

**Comment by Tirokk** [comment]
> [Tirokk](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-28T07:57:25Z" title="Wednesday, August 28th 2019, 9:57:25 am +02:00">Aug 28, 2019</time>_

---


Hey @mattwr18 ,

that's a good point to increase the kubernetes body proxy size. 👍🏼🚀

That works well so far, but I found a lack in functionality which you could solve in this PR or (somebody else) in an other one:

If you place an image bigger than 5m as the teaser image you have an error message. But it appears on the post page. Then afterwards you will not have it in your post.
So I would suggest to not have it in your post on creation if there’s an error. Or?

Bildschirmfoto 2019-08-28 um 09 34 44

Bildschirmfoto 2019-08-28 um 09 35 04

File: deployment/legacy-migration/maintenance-worker/migration/mongo/.env [hunk 1]
```diff
@@ -12,6 +12,6 @@
 # On Windows this resolves to C:\Users\dornhoeschen\AppData\Local\Temp\mongo-export (MinGW)
 EXPORT_PATH='/tmp/mongo-export/'
 EXPORT_MONGOEXPORT_BIN='mongoexport'
-MONGO_EXPORT_SPLIT_SIZE=100
+MONGO_EXPORT_SPLIT_SIZE=4000
```
Comment (Tirokk) [inline deployment/legacy-migration/maintenance-worker/migration/mongo/.env:15]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-28T17:06:16Z" title="Wednesday, August 28th 2019, 7:06:16 pm +02:00">Aug 28, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -12,6 +12,6 @@
 # On Windows this resolves to C:\Users\dornhoeschen\AppData\Local\Temp\mongo-export (MinGW)
 EXPORT_PATH='/tmp/mongo-export/'
 EXPORT_MONGOEXPORT_BIN='mongoexport'
-MONGO_EXPORT_SPLIT_SIZE=100
+MONGO_EXPORT_SPLIT_SIZE=4000
```


code creep - but actually I think that's a better default

**Review by Mogge — APPROVED** [review]
For me this is OK! It's true that `vue-dropzone` should not show a preview picture if the upload is not successful. However, I would assume that people who want to deploy Human Connection would also use our kubernetes configuration.

**Pull request closed without merging**